app/views/experiences/show.html.erb

```diff
@@ -17,7 +17,9 @@
   </div>
   <div class="row">
     <%# Map %>
-    <%#= render @experience.reviews %>
+    <% @experience.reviews.each do |review| %>
+      <%= render './reviews/review', review: review %>
+    <% end %>
```

Comment:
Actually, the commented line from before does exactly what the lines you've added do.
```suggestion
    <%= render @experience.reviews %>
```